src/client/components/home/Home.jsx

```diff
@@ -10,28 +10,35 @@ import './home.css';
 
 const SPINNER_SIZE = 20;
 
-const Home = ({ loading }) => (
+// The spinner is a little taller than it should be,
+// so add 4px to its container's height
+const SEARCH_STATUS_CONTAINER_SIZE = SPINNER_SIZE + 4;
+
+const Home = ({
+  isLoading,
+  searchError
+}) => (
   <div className='home'>
-    <Search />
-    <div className={ classnames({
-      loader: true,
-      hidden: !loading
-    }) }
-    style={{
-      // The loader is a little taller than it should be,
-      // so add 4px to its container's height
-      height: SPINNER_SIZE + 4
-    }}>
-      <ScaleLoader
-        color={ THEME_COLOR_DARK1 }
-        loading={ loading }
-        height={ SPINNER_SIZE }/>
+    <Search hasSearchError={ searchError !== null }/>
+    <div className={ classNames({
```

Comment:
classNames({
       hidden: searchError === null && !isLoading,
       searchStatus: true
     })

is probably complicated enough it would improve readability of the component to extract it to a function